plugins/contivconf/contivconf_api.go

```diff
@@ -17,6 +17,7 @@ import (
 	"net"
```

Comment:
Maybe contivconf_api.go could be moved to api sub-directory? (same with ipnet_api.go)

Reply:
Going back to this - I find it quite confusing now that a subset of contivconf/ipnet APIs (_some_ data structures) are under api/ directory whereas interfaces (and _other_ data structures) remained in the plugin main directory. Dependant plugins may need to import both directories instead of just "api".